prog-o-meter.py

```diff
@@ -115,10 +118,12 @@ def add_day(self):
         Color will be diferent from current progress, to make the new day stand out.
         (Currently the new-day color is hardcoded to be green, but in the future, user should be able to change this themselves).
         """
+        self.days_remaining = self.GOAL - self.days

```

Comment:
I don't think it is necessary to include line 121 and 122 here? They look identical to line 52 and line 53 further up. Since the add_day method belong to the class ProjectGUI(), you should be able to just have line 126, the method will have access to the object's variables self.days_remaining and self.completion_days, which you defined in the __init__ method.

Reply:
I need to re-declare days_remaining and completion_date otherwise the new value isn't calculated when you click the button. It use to work with your solution because you were performing the calculation (self.GOAL - self.days) directly within the "itemconfig()" method

Reply:
ooh, so it is just an update of the value? My bad, that makes sense!

Reply:
I am still working toward correcting this part, once done I'll push all my changes.

Reply:
Cool! :)